homeassistant/components/pyload/sensor.py

```diff
@@ -73,7 +74,7 @@ class PyLoadSensorEntity(StrEnum):
 
 async def async_setup_platform(
     hass: HomeAssistant,
-    config: ConfigType,
+    config: PyLoadConfigEntry,
```

Comment:
This is not a ConfigEntry, this is ConfigType

Reply:
sorry 🙈